README.md

```diff
@@ -10,7 +10,7 @@ We are keeping `packwerk` compatible with current versions of Ruby and Rails, bu
 
 -- _Sandi Metz, Practical Object-Oriented Design in Ruby_
 
-Packwerk is a Ruby gem used to enforce boundaries and modularize Rails applications.
+Packwerk is a Ruby gem used to enforce boundaries and modularize Ruby applications (including Rails apps!).
```

Comment:
```suggestion
Packwerk is a Ruby gem used to enforce boundaries and modularize Rails applications.
```

We have no interest of entering the business of modularize arbitrary Ruby applications. I'm ok to allow load path discovery using Zeitwerk given that makes sense in the context of a Rails application, but if people start to ask for "feature X to support Hanami apps" or "Feature Y to support Sinatra apps" the answer at the moment will be an automatic "sorry, you are on your own", so it is better to not claim in the README that we Packwerk is something it isn't.

Reply:
Makes sense! I know a lot happened today and this PR is no longer relevant for the moment, but I will make sure to have this fixed when we have an acceptable solution in place.